config/cmake/HDF5Macros.cmake

```diff
@@ -70,7 +70,13 @@ macro (H5_SET_VFD_LIST)
       split
       multi
       family
-      splitter
+      # Splitter VFD currently can't be tested with the h5_fileaccess()
```

Comment:
The splitter VFD was previously being tested just with the environment variable approach, where it would add "_wo" to the end of filenames to create the name for the W/O file, which worked will in testing. With the changes in this PR though, any tests that use `h5_fileaccess()` would setup a FAPL with a single W/O path that gets used for any file create or open calls that use that FAPL, leading to failures due to multiple locks on the same file. We should revisit this at some point.